core/src/main/java/nu/yona/server/subscriptions/service/UserService.java

```diff
@@ -277,6 +286,33 @@ private User createUserEntity(UserDto user, UserSignUp signUp)
 		return userEntity;
 	}
 
+	private void addDevicesToEntity(UserDto userDto, User userEntity)
+	{
+		userDto.getOwnPrivateData().getDevices().stream().map(d -> (UserDeviceDto) d).map(d -> createUserDeviceEntity(d, 0))
```

Comment:
Will the device ID always be 0?

Reply:
The initial one. When a device is added, it will get a higher sequence number.

Reply:
I don't see an increment of sequence number anywhere. Maybe good to add a test around it?
Will the user anonymized ID stay the same for current users?

Reply:
Correct. That'll come with [YD-497](http://jira.yona.nu/browse/YD-497).

Reply:
Aha, I see. With that, I can approve this PR.